llvm-spirv/test/extensions/INTEL/SPV_INTEL_joint_matrix/array_of_matrices.ll

```diff
@@ -1,5 +1,5 @@
 ;; Compiled from joint_matrix_bf16_fill_k_cache.cpp from https://github.com/intel/llvm
-;; command: clang++ -fsycl -DSYCL_EXT_ONEAPI_MATRIX_VERSION=4 llvm/sycl/test-e2e/Matrix/joint_matrix_bf16_fill_k_cache.cpp -fsycl-device-only -o test.bc
```

Comment:
@MrSidims i guess we need modify the file in kronos code, right?

Reply:
Yes, please revert the changes here and make them [upstream](https://github.com/KhronosGroup/SPIRV-LLVM-Translator/tree/main). We can move ahead with the intel/llvm changes with or without the removal of these in upstream, right?